infra/recipes/terraform/azure/environments/sandbox/README.md

```diff
@@ -34,6 +34,16 @@ You could set the environment variables for variables to stored terraform backen
 `cp backend.tfvars.template backend.tfvars`.
 2. Edit the backend.tfvars to configure the necessary TF backend state.
 
+## Docker setup
+
+To be able to build the Tables and Housetables service images, you will need to ensure Docker is running on your local machine. This is typically done by opening Docker Desktop. You can check for this by running a command like `docker info` in the terminal.
+
+Sometimes, the images may stop building or uploading to the ACR. This is likely because docker has run out of local memory. To fix this, run `docker system prune --volumes` locally.
+
+### Building OpenHouse project
+
+Before building the images, run `./gradlew clean build` from the root directory. This only works on Java8.
+
 ## Deployment
```

Comment:
Can we update the README with k8s deployment using helm provider?

Reply:
Yes - what more should I include?

Reply:
Yes mainly on the helm chart integration part as part of terraform and the services that are installed.

Reply:
Done